src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/ScenePanelTypeSwitcher.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  SceneComponentProps,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneObjectUrlSyncConfig,
+  SceneObjectUrlValues,
+} from '@grafana/scenes';
+import { RadioButtonGroup } from '@grafana/ui';
+import { reportInteraction } from '@shared/domain/reportInteraction';
+import React from 'react';
+
+export enum PanelType {
+  TIMESERIES = 'time-series',
+  BARGAUGE = 'bar-gauge',
+}
+
+interface ScenePanelTypeSwitcherState extends SceneObjectState {
+  type: PanelType;
+  onChange?: (type: PanelType) => void;
+}
+
+export class ScenePanelTypeSwitcher extends SceneObjectBase<ScenePanelTypeSwitcherState> {
```

Comment:
We might want to DRY and create a new `SceneOptionsSwitcher` that could replace this Scene object and `SceneLayoutSwitcher`.